Base/res/ladybird/about-pages/settings.html

```diff
@@ -379,6 +388,46 @@ <h1>Ladybird Settings</h1>
             </div>
         </div>
 
+        <div class="card">
+            <div class="card-header">Network</div>
+            <div class="card-body">
+                <div class="card-group">
+                    <div class="card-header header-level-2">
```

Comment:
Let's add `inline-container` to this class list. It'll justify the forcibly-enabled text to the right. Just to match the autoplay setting:
![428400014-a5de03aa-8eee-4234-bf5e-575922df474c](https://github.com/user-attachments/assets/671d4adf-d4fa-462c-a3bb-0aab7582e6ab)